styles/index.styles.ts

```diff
@@ -68,4 +70,26 @@ export const AppWrapper = styled.div`
 export const Content = styled.div`
   margin: 30px;
   display: flex;
+  flex-wrap: wrap;
+  justify-content: space-evenly;
+
+  background: #dae0e6;
+
+  ${media.bigDesktop} {
```

Comment:
The way we write media queries here is different now that we changed media.ts, but looks fine to me :)